packages/@webex/plugin-cc/src/constants.ts

```diff
@@ -13,3 +13,10 @@ export const AQM_REQS_FILE = 'aqm-reqs';
 export const TASK_MANAGER_FILE = 'TaskManager';
 export const AGENT_STATE_CHANGE = 'agent:stateChange';
 export const AGENT_MULTI_LOGIN = 'agent:multiLogin';
+// AGENT OUTDIAL CONSTANTS
+export const OUTDIAL_DIRECTION = 'OUTBOUND';
+export const ATTRIBUTES = {};
+export const OUTDIAL_MEDIA_TYPE = 'telephony';
+export enum OutboundType {
```

Comment:
Hmmm, I think let's go ahead with `string` only, but let's restrict it to certain ones only.
For eg.
```suggestion
export const OUTBOUND_TYPE = 'OUTDIAL' | <we can add more if needed>
```